src/constants/sites.ts

```diff
@@ -0,0 +1,32 @@
+export const TESTING_SITES = [
+  "a-test-v4",
+  "alex-debug-staging",
+  "alexis-test-abc",
+  "alexis-test-bcd",
+  "chin-test",
+  "harish-test-site",
+  "jikun-test",
+  "kishore-test",
+  "kishore-test-dev-gh",
+  "kishore-test-dev-emil",
+  "kishore-test-email-login",
+  "kishore-test-test-email-flow",
+  "rcd-test-31",
+  "rcd-test-repo",
+  "rcd-test-repo2",
+  "rcd-test-repo3",
+  "test-jacksontest",
+  "zhongjun-test",
+  "zhongjun-test-amplify",
+  "zhongjun-test-emailam",
+]
+
+export const OGP_SITES = [
```

Comment:
what if we need to rollout to more sites maybe a few agencies? we need a hotfix rollout like for ggs right? Not sure if this could be a possibility. do clarify if it is a possibility, and if so let's move to env var. Else, fine with this temp approach

Reply:
to be clear here - i **want** this to be a constant in code, rather than an env var. i'll outline my reasoning below

env vars are primarily useful for _ad-hoc_ updates - when we want to adjust things on the fly without triggering a full deploy. this is useful for minor features (subjective but ok) where we don't really care about having, for instance, tracking values between rollouts. this, however, does introduce overhead to eg: the release process where the oncall needs to update said values.

having a constant in code means that our rollout is tied to our weekly releases. this trade off is acceptable to me for a **phased** rollout, where the phase are going to be in weeks anyway (ie, i release to 10% this week, 20% next week for example). additionally, it's very explicit and clear about what's gonna get the new features and we get things liek type checking, git history etc so it's even more clear. 

me relying on constants also rests on the assumption that growthbook/ld is going to be in either this week or the next - this is just a short lived measure for clarity that won't be here in a few weeks.